state/connection/QueryRunner.kt

```diff
@@ -214,7 +217,13 @@ class QueryRunner constructor(
         noResultMsg = MATCH_QUERY_NO_RESULT,
         queryStr = query.toString(),
         stream = Response.Stream.ConceptMaps(MATCH)
-    ) { transaction.query().match(query) }
+    ) { if (query.modifiers().limit().isPresent) {
+            transaction.query().match(query)
+        } else {
+            val queryWithLimit = TypeQLMatch.Limited(query, preferenceMgr.queryLimit)
+            transaction.query().match(queryWithLimit)
+        }
+     }
```

Comment:
Is it sufficient to have this limit for match, or should match aggregate or match aggregate groups have this too?

Reply:
Definitely not useful for match aggregates - the answer is a single number! No graph output either.

As for match groups and match group aggregates, I'm not entirely sure to be honest. Does match group produce a graph output? I think it doesn't, right?

Also, I'm not entirely sure how the limit on a match group / match group aggregate works - we should clarify and then reach a decision.

Reply:
They don't produce graph outputs - so our implementation is correct as is.